api.js

```diff
@@ -1,9 +1,17 @@
 var mongo = require('mongodb').MongoClient,
-    fiddles = null;
+    fiddles = null,
+    mongoConnectionStr = process.env.MONGOHQ_URL ?
```

Comment:
I would feel better about always reading this from an environment variable. I would be up for changing the environment variable to something like `ES6FIDDLE_DB_URL` or something similar.